browser/profiles/brave_profile_manager.cc

```diff
@@ -103,7 +103,6 @@ void BraveProfileManager::DoFinalInitForServices(Profile* profile,
   DCHECK(status);
   status->UpdateGCMDriverStatus();
 #endif
-  brave_news::BraveNewsControllerFactory::GetForContext(profile);
```

Comment:
maybe we could delete `#include "brave/browser/brave_news/brave_news_controller_factory.h"` also?